e3nn/tensor_product.py

```diff
@@ -73,39 +73,70 @@ def forward(self, features_1, features_2):
 
 
 class LearnableTensorProduct(torch.nn.Module):
-    def __init__(self, Rs_mid_1, Rs_mid_2, mul_mid, Rs_out, selection_rule=o3.selection_rule):
+    def __init__(self, Rs_mid_1, Rs_mid_2, Rs_out,
```

Comment:
What about `in`? `mid` means middle?
```suggestion
    def __init__(self, Rs_in1, Rs_in2, Rs_out,
```